Lares/Assets/Mechanics/Inventory/Runtime/InventoryManager.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Lares.Inventory
+{
+    public class InventoryManager : MonoBehaviour
+    {
+        public GameObject PlayerReference;
+        public List<InventoryNode> InventoryData;
+        public ScriptableInventory InventoryList;
+
+        public bool IsInInventory(Guid id) => InventoryData.Any(i => new Guid(i.Item.ItemData.ItemId) == id);
+        public bool IsValidID(Guid id) => InventoryList.ItemList.Any(i => new Guid(i.ItemData.ItemId) == id);
+        public InventoryItem GetInventoryItemById(Guid id) => InventoryList.ItemList.Find(i => new Guid(i.ItemData.ItemId) == id);
+
+        private void OnEnable()
+        { 
+            if (InventoryList.ItemList.Count == 0) { Debug.Log("InventoryList is empty!"); }
+            InventoryData = new();
```

Comment:
Missing type specifier, as the type is not visible on the line.
```suggestion
            InventoryData = new List<InventoryNode>();
```